app/Filament/Admin/Resources/Eggs/Pages/EditEgg.php

```
@@ -50,36 +51,58 @@ public function form(Schema $schema): Schema
+                            FileUpload::make('image')
+                                ->hiddenLabel()
+                                ->avatar()
+                                ->alignCenter()
+                                ->imagePreviewHeight('50')
+                                ->previewable()
+                                ->openable(false)
+                                ->downloadable(false)
+                                ->maxSize(1024)
+                                ->maxFiles(1)
+                                ->columnSpan(['default' => 2, 'sm' => 2, 'md' => 1, 'lg' => 1])
+                                ->saveUploadedFileUsing(function ($file, $state, $record) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$record'. (undefined)

(UnusedFormalParameter)